r2/r2/models/builder.py

```diff
@@ -199,9 +218,6 @@ def wrap_items(self, items):
                     link="/user/%s" % w.author.name,
                 )
 
-            if hasattr(item, "sr_id") and item.sr_id is not None:
-                w.subreddit = subreddits[item.sr_id]
-
```

Comment:
Had to be moved up to L180 because underneath that is where the attribute is inserted.